CHANGE_LOG

```diff
@@ -1,3 +1,83 @@
+Version 0.42.0
+--------------
+
+In this release the major features are:
+
+- The capability to launch and attach the GDB debugger from within a jitted
+  function.
+- The upgrading of LLVM to version 7.0.0.
+
+Here are some enhancements from contributed PRs:
```

Comment:
I really like this section calling out community contributions!

Reply:
With the increase in contributed PRs, I feel like we should start doing that to encourage more contributions.

Reply:
It's a great idea.